tools/update-tools/update_tools.py

```diff
@@ -1056,12 +1056,22 @@ def CleanUpdateFiles(self):
           os.path.join(self.fota_sdcard, "updates", "fota")
         ]
 
+        # Device-specific files that we need to cleanup
+        # Should be specified as a space-separated list of files in
+        # the env variable FOTA_DEVICE_DATA_FILES
```

Comment:
I'd prefer if we added an argument like --fota-cleanup-files, we use environment variables in the makefile but not so much in this script and maybe it's better this way.